ray_ci/bootstrap_linux.sh

```diff
@@ -12,6 +12,11 @@ fi
 
 # Commit message instructions
 
+if [[ "$BUILDKITE_MESSAGE" =~ "[early_kickoff]" ]]; then
```

Comment:
@krfricke just for my understanding, this must be part of my commit message?

Reply:
Yes! It's always the last commit message in the PR